packages/hyperverse-ethereum/source/Provider.tsx

```diff
@@ -7,9 +7,9 @@ import {
 } from "react";
 import Web3Modal from "web3modal";
 import WalletConnectProvider from "@walletconnect/web3-provider";
-import { providers } from "ethers";
+import { providers, ethers } from "ethers";
 
-const INFURA_ID = process.env.NEXT_PUBLIC_ALCHEMY_API_KEY!;
+const INFURA_ID = 'fb9f66bab7574d70b281f62e19c27d49';
```

Comment:
What account is this Infura key under? If it's your personal account we should create a team account so it can be managed

We will allow users to use our key by default but they would be able to switch it out for their own